components/dashboard/src/prebuilds/detail/PrebuildDetailPage.tsx

```diff
@@ -103,7 +103,14 @@ export const PrebuildDetailPage: FC = () => {
     }, [isTriggerError, triggerError, toast]);
 
     const prebuildPhase = useMemo(() => {
-        const name = currentPrebuild?.status?.phase?.name;
+        if (!currentPrebuild) {
+            return {
+                icon: <LoadingState size={20} delay={false} />,
+                description: "",
+            };
+        }
+
+        const name = currentPrebuild.status?.phase?.name;
```

Comment:
This is not a blocking comment. If we want to give it a name, named `phase` or `phaseName` maybe better